compiler/src/iree/compiler/Codegen/LLVMCPU/KernelDispatch.cpp

```diff
@@ -1127,6 +1127,11 @@ static SmallVector<int64_t> getLinalgExtDefaultWorkgroupTileSizes(
   return workgroupTileSizes;
 }
 
+static bool isPackMatmulLHS(tensor::PackOp op) {
+  return op.getSourceRank() == 2 && op.getInnerDimsPos().size() == 2 &&
+         op.getInnerDimsPos()[0] == 0 && op.getInnerDimsPos()[1] == 1;
+}
+
```

Comment:
Would it make sense to move this to a header so that you can also reuse in LLVMCPULowerExecutableTarget?